tests/cache/models/samples/upsertBulkCaseSensitive.js

```diff
@@ -73,27 +74,29 @@ describe('tests/cache/models/samples/upsertBulkCaseSensitive.js, ' +
     });
   });
 
-  it('new sample: different case name should NOT modify sample name', (done) => {
+  it('new sample: different case name should NOT modify sample name',
+  (done) => {
     const path = '/v1/samples/upsert/bulk';
     const sampleName = '___Subject1|___Aspect2';
 
     api.post(path)
     .set('Authorization', token)
     .send([{ name: sampleName.toLowerCase(), value: '6' }])
-    .then(() => {
+    .then((err, res) => {
```

Comment:
is the err and res necessary here?

Reply:
nope, I had added them so I could drill into where the bug was, but I'll pull them out now